src/__tests__/visualizer/algorithm/sortingalgorithms/mergeSort.test.js

```diff
@@ -1,19 +1,24 @@
 import mergeSort from '../../../../visualizer/algorithm/sortingalgorithms/mergeSort';
+import { assertSortSuccess } from '../../../../_testutil/TestUtil';
+import {
+  ARRAY_DUPLICATES,
+  ARRAY_EXPECTED_RANDOM_MIXED,
+  ARRAY_EXPECTED_RANDOM_NEGATIVE,
+  ARRAY_EXPECTED_RANDOM_POSITIVE,
+  ARRAY_EXPECTED_STRICTLY_DESCENDING,
+  ARRAY_RANDOM_MIXED,
+  ARRAY_RANDOM_NEGATIVE,
+  ARRAY_RANDOM_POSITIVE,
+  ARRAY_STRICTLY_ASCENDING,
+  ARRAY_STRICTLY_DESCENDING,
+  EMPTY_ARRAY,
+} from '../../../../_testutil/ArraysUtil';
 
 test('Empty Array', () => {
-  const arr = [];
-  const expected = [];
-  expect(mergeSort(arr)).toEqual(expected);
+  assertMergeSortSuccess(EMPTY_ARRAY, EMPTY_ARRAY, EMPTY_ARRAY);
 });
 
 test('Already Sorted with duplicates', () => {
-  const arr = [
-    { id: 1, height: 3, isShift: false, xDirection: 0, pos: 0, prevPos: 0 },
-    { id: 2, height: 3, isShift: false, xDirection: 10, pos: 1, prevPos: 1 },
-    { id: 3, height: 3, isShift: false, xDirection: 20, pos: 2, prevPos: 2 },
-    { id: 4, height: 3, isShift: false, xDirection: 30, pos: 3, prevPos: 3 },
-  ];
-  const actual = mergeSort(arr);
   const expected = [
```

Comment:
The expected array for duplicates case and ascending case looks the same? Might want to create a global array for both to use.